lib/trebuchet/backend/redis_hammerspaced.rb

```diff
@@ -0,0 +1,98 @@
+require 'trebuchet/backend/redis'
+require 'json'
+
+class Trebuchet::Backend::RedisHammerspaced < Trebuchet::Backend::Redis
+
+  # This class will rely on a cron job to sync all trebuchet features
+  # to local hammerspace thus this class never directly updates hammerspace
+  # We also cache in memory the features as we do in redis_cached
+
+  attr_accessor :namespace
+
+  def initialize(*args)
+    @namespace = 'trebuchet/'
+    begin
+      # args must be a hash
+      @options = args.first
+      @redis = @options[:client]
+      @hammerspace = @options[:hammerspace]
+      unless @options[:skip_check]
+        # raise error if not connectedUncaught ReferenceError: google is not defined
```

Comment:
It's there so in production we won't skip check. That way we will error out early during rails initialization.